core/api/src/services/svix/index.ts

```diff
@@ -31,15 +31,36 @@ function prefixObjectKeys(
   }, {})
 }
 
-export const CallbackService = (config: SvixConfig) => {
+interface CallbackService {
+  sendMessage: (args: {
+    eventType: string
+    accountId: AccountId
+    walletId: WalletId
+    payload: Record<string, JSONValue>
+  }) => Promise<ApplicationError | true>
+  getPortalUrl: (accountId: AccountId) => Promise<ApplicationError | string>
```

Comment:
ApplicationError is from app layer, please create CallbackServiceError `export class CallbackServiceError extends CallbackError {}` here https://github.com/GaloyMoney/galoy/blob/main/core/api/src/domain/callback/errors.ts